### PR TITLE
Clean paths Fastfile

### PR DESCRIPTION
**Pull request by pablog**
Removed path formatters from `Fastfile`.

The idea is to clean up this file removing paths formatters, and hash constants to a different place, and split `Fastfile` in different files, since it's starting to get too big.

My idea is to have a `Fastfile.private` with the private lanes, and a `Fastfile`, with the public lanes using the private ones.

### CODE REVIEW

File: Fastfile [hunk 1]
```diff
@@ -169,7 +163,7 @@ reflected in `Info.plist` during building."
 
       desc "Get rollbar server access token from configuration file."
       rollbar_server_access_token = read_xcconfig_property(
-        xcconfig_path: project_xcconfig_path % [project_name, build_configurations[build_configuration]],
+        build_configuration: build_configurations[build_configuration],
```
Comment (damfinkel) [inline Fastfile]: you changed the key you use here, this works the same way as before?

Comment (pablog) [reply Fastfile]: Yes, the idea is to provide the `build_configurations`, and let the action build the path.

File: Fastfile [hunk 2]
```diff
@@ -307,7 +272,7 @@ reflected in `Info.plist` during building."
       app_identifier: bundle_identifier,
       force: false,
       p12_password: p12_password,
-      output_path: pem_output_path % build_configuration.to_s
+      output_path: "#{build_configuration.to_s}_push_certificates"
```
Comment (danielaRiesgo) [inline Fastfile]: I don't like this change. I prefer a constant that can be easily identified and changed.

File: actions/read_xcconfig_property.rb [hunk 3]
```diff
@@ -8,12 +8,18 @@ class ReadXcconfigPropertyAction < Action
       # This is useful in case sensitive data stored in xcconfig files
       # is needed to be accessed from a fastlane script.
 
+      # Default xcconfig files path.
+      PROJECT_XCCONFIG_PATH = "%s/ConfigurationFiles/%s.xcconfig"
+
       def self.run(params)
-        if !File.exist?(params[:xcconfig_path])
+        xcconfig_path = PROJECT_XCCONFIG_PATH % [params[:project_name], params[:build_configuration]]
```
Comment (danielaRiesgo) [inline actions/read_xcconfig_property.rb]: But you are not sending `project_name` parameter in here: https://github.com/Wolox/fastlane-mobile/pull/3/files#diff-c41a4df45a632f3c6b44719dcfef8883L171

is that okay?

Comment (danielaRiesgo) [reply actions/read_xcconfig_property.rb]: EDIT: Just saw there is a default value in self.available_options

**Review by danielaRiesgo — APPROVED** [review]

**Review by dtilve — APPROVED** [review]
👍

File: Fastfile (hunk 2 above)
Comment (pablog) [reply Fastfile]: I agree.

Please check this commit, where I extracted the `output_path` to a property inside the lane:

https://github.com/Wolox/fastlane-mobile/commit/fc05035ec22bdc082deb9a7a9c6e33f34b2d1881#diff-b7412e7e00778b956edd9dc6cc704c47R165

**Pull request merged**